CONTRIBUTING.md

```diff
@@ -32,9 +56,54 @@ Please take a look at the current open issues to see if any of them interest you
 5. Describe the problem and add references to the related issues that the request solves.
```

Comment:
"addresses" instead of "solves".  (not all issues are bugs, nor do all PRs completely resolve an issue.)